single_pages/dashboard/system/environment/security_header_extended.php

```diff
@@ -91,6 +92,22 @@
         </div>
     </div>
 
+    <div class="form-group">
+        <?= $form->label('', t('X-Content-Type-Options')) ?>
+        <div class="form-check">
+            <?= $form->checkbox('nosniff', '1', $nosniff) ?>
+            <?= $form->label('nosniff', t('Enable X-Content-Type-Options (nosniff)')) ?>
```

Comment:
@SashaMcr 

nosniff is the value for x-content-type-options header. not an alternative name of the header.